layouts/partials/compliance/insights.html

```diff
@@ -0,0 +1,39 @@
+<section class="mt-8 px-4 md:px-0">
+    <div class="container mx-auto max-w-4xl">
+        <h3>Pulumi Insights</h3>
```

Comment:
I think you should do some sort of divide between here where we talk about Pulumi and the stuff above. Maybe its doing this section in more of that grey color?